website/docs/components/pagination/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,40 @@
+## Conformance rating
+
+<!-- Update conformance rating badge with correct status and remove the others -->
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
+
+<Doc::Badge @type="warning">Conditionally conformant</Doc::Badge>
+
+Explain how the component is conditionally conformant. For example, any component that offers text truncation is conditionally conformant- it is conformant as long as the text truncation feature is not used.
```

Comment:
```suggestion
Explain how the component is conditionally conformant. For example, any component that offers text truncation is conditionally conformant—it is conformant as long as the text truncation feature is not used.
```

Reply:
@KristinLBradley I'll disregard these comments as this content comes from the template and I believe it's mostly informational. If it needs to be updated, we should update the template directly.